spec/jobs/ra/fetch_changes_batch_job_spec.rb

```diff
@@ -180,13 +216,18 @@
 
       district_change = Ra::DistrictChange.first
       expect(district_change).to have_attributes(
+        change_id: nil,
         district_id: 4764,
         municipality_id: 92,
         database_operation: 'INSERT',
         version_id: 1,
         created_reason: 'CREATE',
         valid_to: Time.parse('2004-04-30T23:59:59+02:00'),
-        unique_numbering: false
+        unique_numbering: false,
+
+        change: nil,
+        district_object: Ra::District.find(4764),
+        municipality: Ra::Municipality.find(92),
```

Comment:
Aha, uz chapem terajsi `change_id` je vlastne `objectId` z xmlka? rozmyslam ci tam nedame nieco ako `object_identifier` kedze to nie je cudzi kluc nikam, je to nejake ich id. Tym by sa vyriesila *nasty* kolizia s Ruby `object_id`